create_aio_app/template/{{cookiecutter.project_name}}/requirements/development.txt

```diff
@@ -1,10 +1,10 @@
--e .
+-r production.txt
 
-mypy==0.641
-flake8==3.6.0
-pytest==4.0.1
-pytest-cov==2.6.0
-pytest-aiohttp==0.3.0
+mypy
```

Comment:
`Freezing Dependency` its means that you hardcode version of your dependency.
It's good practice :) So please return versions in  `development.txt` and add in `production.txt`

Reply:
@Arfey 
At work, we use the  skeleton packages without versions, when starting a new project, we freeze the latest versions of packages.
Or do you have code for specific versions?